src/foam/u2/TableView.js

```diff
@@ -133,7 +138,26 @@ foam.CLASS({
 
         e = e.start('tr');
         for ( var i = 0 ; i < properties_.length ; i++ ) {
-          e = e.start('td').add(properties_[i].label).end();
+          var sorting$ = this.sortOrder$.map(function(prop, order) {
```

Comment:
At this point we should refactor the header into its own model, this is getting too large for my tastes.